### PR TITLE
Update sk_SK.lang

### PR DESCRIPTION
**Pull request by leSamo**

### CODE REVIEW

File: src/main/resources/assets/bcadditions/lang/sk_SK.lang [hunk 1]
```diff
@@ -11,66 +11,429 @@ tile.blockDusterKinetic.name=Kinetický drvič
...
+tooltip.colorSortingMode=Triediaci mód
+tooltip.colorSortingMode.info=Zafarbí triediace časti diazuritovej a lazuritovej rúry.
+tooltip.defaultColor=Štandardná farba
+tooltip.compressor.fill=Naplniť
+tooltip.compressor.empty=Vyprázdniť
+tooltip.none=Žiadne
+tooltip.forKineticMultiTool=Použiteľné s kinetickým multinástrojom
+tooltip.forUpgradableMachines=Použiteľné s prístrojmi, ktoré sa dajú vylepšiť
+tooltip.stick.0=Najlepšie funguje %s čipom
+tooltip.stick.1=%s tyč pre inštaláciu
```
Comment (leSamo) [inline src/main/resources/assets/bcadditions/lang/sk_SK.lang]: I had to be a bit creative here, because my language has double negation. But it works.

**Comment by iTitus** [comment]
Thank you for that ;)
If you think that some phrases are hard to translate (especially regarding those %s things) let us know, we will try our best to fix that.

**Comment by leSamo** [comment]
It's cool. Also, the stick part reminded me why I don't want to translate metallurgy, whoa.

**Comment by iTitus** [comment]
Oh yeah, it's a lot of repetition...

**Comment by AEnterprise** [comment]
thanks for the translation

**Comment by leSamo** [comment]
No, repetition is okay, I've got stuff to handle that. I mean all the alloys. They are hard to translate.

**Pull request merged**